lambda_backend/predict_microservice/base_inference.py

```diff
@@ -0,0 +1,144 @@
+import cv2
```

Comment:
Can use symlinks to avoid having two of the exact same files:
https://mokacoding.com/blog/symliks-in-git/